### PR TITLE
Adding Circle Chart and ShadCN

### LINKED ISSUE #2: build circle chart

### PR DESCRIPTION
**Pull request by fadyfawzydev**

### CODE REVIEW

File: app/_components/chartItemsListing/ChartItemsListing.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+const ChartItemsListing: React.FC<ChartItemsListingProps> = ({
+  chartData,
+  className = "flex w-full flex-col gap-4",
+}) => {
+  return (
+    <div className={className}>
+      {chartData.map((item, ndx) => (
+        <div className="flex items-center justify-between gap-4" key={ndx}>
+          <div className="flex items-center gap-4">
+            <div className="h-2 w-2 rounded-full" style={{ backgroundColor: item.color }}></div>
```
Comment (MennaTallah2) [inline app/_components/chartItemsListing/ChartItemsListing.tsx:17]: can we do this with before ?

File: graph.svg
Comment (MennaTallah2) [inline graph.svg:1]: what are the changes here?

File: package.json [hunk 2]
```diff
@@ -50,11 +50,14 @@
...
     "clsx": "^2.1.0",
     "lodash": "^4.17.21",
+    "lucide-react": "^0.364.0",
     "next": "^14.0.3",
     "next-compose-plugins": "^2.2.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "tailwind-merge": "^2.0.0",
+    "recharts": "^2.12.3",
+    "tailwind-merge": "^2.2.2",
```
Comment (MennaTallah2) [inline package.json:59]: Do we need those libraries or they're  dependancies ?

**Review by MennaTallah2 — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (fadyfawzydev) [reply package.json:59]: These are dependencies for shadcn

**Pull request merged**